plugins/healthcheck/healthz.go

```diff
@@ -29,9 +29,10 @@ func init() {
 }
 
 // HealthChecker checks if current server is healthy.
-type HealthChecker func(ctx context.Context) error
+// The `checkType` parameter indicates the type of health check, such as liveness or readiness.
+type HealthChecker func(ctx context.Context, checkType string) error
```

Comment:
This change breaks the backward compatibility.

Reply:
Yes, I didn't find a better way to keep the backward compatibility. Any suggestion?